src/array.jl

```diff
@@ -15,12 +15,23 @@ _indices(x::OneHotArray) = x.indices
 _indices(x::Base.ReshapedArray{<: Any, <: Any, <: OneHotArray}) =
   reshape(parent(x).indices, x.dims[2:end])
 
+"""
+    OneHotVector{T, L} = OneHotArray{T, L, 0, 1, T}
+
+Constructed by [`onehot`](@ref). Parameter `T` is the type of the underlying
+storage and `L` is the length of labels.
```

Comment:
```suggestion
A one-hot vector (with `L` labels) typically constructed using [`onehot`](@ref).
Stored efficiently as a single index of type `T`.
```

Reply:
Maybe this one too could clarify that L is the length, 

```suggestion
A one-hot vector with `L` labels (i.e. `length(A) == L` and `count(A) == 1`) typically constructed by [`onehot`](@ref).
Stored efficiently as a single index of type `T`, usually `UInt32`.
```